yalexs/api_common.py

```diff
@@ -126,7 +127,7 @@ def _activity_from_dict(source, activity_dict):
         return DoorbellImageCaptureActivity(source, activity_dict)
     if action in ACTIVITY_ACTIONS_DOORBELL_VIEW:
         return DoorbellViewActivity(source, activity_dict)
-    if action in ACTIVITY_ACTIONS_LOCK_OPERATION:
+    if action in ACTIVITY_ACTIONS_LOCK_OPERATION and user:
```

Comment:
User isn't always set.  I don't think this change is safe as we could miss operations.